src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -262,7 +264,9 @@ internal interface ISyntaxFactsService : ILanguageService
         bool IsDirective(SyntaxNode node);
         bool IsForEachStatement(SyntaxNode node);
         bool IsLockStatement(SyntaxNode node);
-        bool IsUsingStatement(SyntaxNode node);
+#nullable enable
```

Comment:
this is just getting messy and unpleasant IMO.  just come and do this full file at some point the future